code/game/objects/effects/portals.dm

```diff
@@ -1,35 +1,102 @@
 /obj/effect/portal
 	name = "portal"
-	desc = "Looks unstable. Best to test it with the clown."
+	desc = "There's a hole in the sky... through which I can fly."

```

Comment:
Yeah but still, the first one is actually ss13 related, which trumps random reference to portal